test/dag_benchmarks.cc

```diff
@@ -61,77 +61,24 @@ TEMPLATE_TEST_CASE_SIG("benchmark intrusive graph dependency handling with N nod
 }
 
 TEST_CASE("benchmark task handling", "[benchmark][group:task-graph]") {
-	using namespace std::chrono_literals;
-	std::unique_ptr<task_manager> tm;
-
 	constexpr int N = 10000;
 	constexpr int report_interval = 10;
 
-	std::atomic<int> highest_tid = 0;
-	std::atomic<int> highest_tid_to_delete = 0;
-
-	auto initialization_lambda = [&] {
-		highest_tid = 0;
-		tm = std::make_unique<task_manager>(1, nullptr);
+	BENCHMARK("generating and deleting tasks") {
```

Comment:
Due to the loss of a section, this will probably no longer be automatically compared by the benchmark eval script, so we need to manually look at that comparison.

Reply:
It's over 20% slower than before, which is a bit sad given that it's simpler.